src/Components/OATGraphViewer/Internal/OATGraphCustomNode.tsx

```diff
@@ -171,20 +171,22 @@ const OATGraphCustomNode: React.FC<IOATGraphCustomNodeProps> = ({
 
     return (
         <>
-            <Handle
-                type="target"
-                position="top"
-                className={graphViewerStyles.handle}
-                isConnectable={isConnectable}
-            />
+            {data.type === OATUntargetedRelationshipName && (
```

Comment:
Why are we having this handle just for the Untargeted Relationship? I'm aware we stop needing it since you are reading the target position on the Custom edge from the state instead the props but why still need it in this case.